backends/aerospike.go

```diff
@@ -50,22 +50,29 @@ func (a *Aerospike) Get(ctx context.Context, key string) (string, error) {
 	return rec.Bins[binValue].(string), nil
 }
 
-func (a *Aerospike) Put(ctx context.Context, key string, value string, ttlSeconds int) error {
-	asKey, err := as.NewKey(a.cfg.Namespace, setName, key)
-	if err != nil {
-		return err
-	}
-	if ttlSeconds == 0 {
-		ttlSeconds = a.cfg.DefaultTTL
-	}
-	bins := as.BinMap{
-		binValue: value,
-	}
-	err = a.client.Put(&as.WritePolicy{
-		Expiration: uint32(ttlSeconds),
-	}, asKey, bins)
-	if err != nil {
-		return err
+func (a *Aerospike) MultiPut(ctx context.Context, payloads []Payload) error {
+	for _, payload := range payloads {
+		key := payload.Key
+		value := payload.Value
+		ttlSeconds := payload.TtlSeconds
+
+		asKey, err := as.NewKey(a.cfg.Namespace, setName, key)
+		if err != nil {
+			return err
+		}
+		if ttlSeconds == 0 {
+			ttlSeconds = a.cfg.DefaultTTL
+		}
+		bins := as.BinMap{
+			binValue: value,
+		}
+		err = a.client.Put(&as.WritePolicy{
+			Expiration: uint32(ttlSeconds),
+		}, asKey, bins)
+		if err != nil {
+			return err
```

Comment:
Pardon my ignorance, but I noticed that the only return statement in the for loop is for the error case. What happens if there's an error before the last iteration of the for loop? Does execution continue for the other put requests?

Reply:
It errors out if a put fails and does not continue processing the other requests. This is the same as the previous behavior.